src/Sylius/Bundle/WebBundle/Resources/views/Backend/Macros/misc.html.twig

```diff
@@ -72,8 +81,19 @@
     <form action="{{ path('sylius_backend_payment_update_state', {'id': payment.id, 'transition': transition}) }}" method="post" style="display: inline">
         <input type="hidden" name="_method" value="PUT">
         <button class="btn btn-{{ transition == 'cancel' ? 'warning' : 'primary' }}" type="submit">
-            <i class="glyphicon glyphicon-{{ transition == 'cancel' ? 'remove' : 'share' }}"></i> <span>{{ transition }}</span>
+            <i class="glyphicon glyphicon-{{ transition == 'cancel' ? 'remove' : 'share' }}"></i> <span>{{ ('sylius.payment.transition'~transition)|trans }}</span>
         </button>
     </form>
 {%- endfor %}
 {% endmacro %}
+
+{% macro shipment_form(shipment) %}
+{% for transition in ['hold', 'release', 'backorder', 'prepare', 'ship', 'cancel', 'return'] if sm_can(shipment, transition, 'sylius_shipment') -%}
+    <form action="{{ path('sylius_backend_shipment_update_state', {'id': shipment.id, 'transition': transition}) }}" method="post" style="display: inline">
+        <input type="hidden" name="_method" value="PUT">
+        <button class="btn btn-{{ transition == 'cancel' ? 'warning' : 'primary' }}" type="submit">
+            <i class="glyphicon glyphicon-{{ transition == 'cancel' ? 'remove' : 'share' }}"></i> <span>{{ ('sylius.shipment.transition')~transition|trans }}</span>
```

Comment:
`{{ ('sylius.shipment.transition.'~transition)|trans }}` You missed the last dot and wrong parenthesis position ;)